File: docs/components_page/components/dropdown/direction.jl
using DashBootstrapComponents

items = [
    dbc_dropdownmenuitem("First"),
    dbc_dropdownmenuitem(divider = true),
    dbc_dropdownmenuitem("Second"),
];

dropdown = dbc_row(
    [
        dbc_col(
            dbc_dropdownmenu(
                label = "Dropdown (default)",
                children = items,
                direction = "down",
            ),
            width = "auto",
        ),
        dbc_col(
            dbc_dropdownmenu(label = "Dropstart", children = items, direction = "start"),
            width = "auto",
        ),
        dbc_col(
            dbc_dropdownmenu(label = "Dropend", children = items, direction = "end"),
            width = "auto",
        ),
        dbc_col(
            dbc_dropdownmenu(label = "Dropup", children = items, direction = "up"),
            width = "auto",
        ),
    ],
    justify = "between",
);

File: docs/components_page/components/dropdown/direction.R
library(dashBootstrapComponents)

items_direction <- list(
  dbcDropdownMenuItem("First"),
  dbcDropdownMenuItem(divider = TRUE),
  dbcDropdownMenuItem("Second")
)

dropdown <- dbcRow(
  list(
    dbcCol(
      dbcDropdownMenu(
        label = "Dropdown (default)", children = items_direction,
        direction = "down"
      ),
      width = "auto"
    ),
    dbcCol(
      dbcDropdownMenu(
        label = "Dropstart", children = items_direction, direction = "start"
      ),
      width = "auto"
    ),
    dbcCol(
      dbcDropdownMenu(
        label = "Dropend", children = items_direction, direction = "end"
      ),
      width = "auto"
    ),
    dbcCol(
      dbcDropdownMenu(
        label = "Dropup", children = items_direction,
        direction = "up"
      ),
      width = "auto"
    )
  ),
  justify = "between"
)
